db/migrate/20180508162616_create_builds.rb

```diff
@@ -0,0 +1,27 @@
+# 20180508162616_create_builds.rb
+
+# rubocop:disable all
+class CreateBuilds < ActiveRecord::Migration
+  DOWNTIME = false
+
+  def up
+    create_table :builds do |t|
+      t.integer :namespace_id
+      t.integer :project_id
+      t.integer :active_build_id
+      t.string :status
+      t.string :file_path
+      t.string :comment
+      t.string :commmit
+      t.string :branch
+      t.datetime :ts, null: false
```

Comment:
Should we use `created_at` instead? Or do we want to stick with the current naming convention?

Reply:
Going to update it to created_at

Reply:
Also added an updated_at property

Reply:
Builds are immutable, but whatever can't hurt.